test/unit_test/dsjson_parser_test.cc

```diff
@@ -440,7 +440,7 @@ BOOST_AUTO_TEST_CASE(parse_dsjson_slates)
       label6.probabilities, std::vector<ACTION_SCORE::action_score>{{1, 0.8f}}, FLOAT_TOL);
   const auto& label7 = examples[7]->l.slates;
   check_collections_with_float_tolerance(
-      label7.probabilities, std::vector<ACTION_SCORE::action_score>{{0, 0.6f}}, FLOAT_TOL);
+      label7.probabilities, std::vector<ACTION_SCORE::action_score>{{0, 0.6f}, {1, 0.4f}}, FLOAT_TOL);

```

Comment:
was this just a general bug of not working properly?

Reply:
I extended the test so it would have caught this issue.